Tests/Foundation/Tests/TestTimeZone.swift

```diff
@@ -160,7 +160,12 @@ class TestTimeZone: XCTestCase {
         var lt = tm()
         localtime_r(&t, &lt)
         let zoneName = NSTimeZone.system.abbreviation() ?? "Invalid Abbreviation"
-        let expectedName = String(cString: lt.tm_zone, encoding: .ascii) ?? "Invalid Zone"
```

Comment:
Again, why not [force unwrap instead](https://github.com/apple/swift-corelibs-foundation/pull/4850/files#diff-0009d42a85c11aa871598697f5afaa46e1639c92bb890bdfe303c989b398a16cR163)?